examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -82,7 +91,7 @@ function App() {
                     <div className="fill">
                         <Routes>
                             {window.location.origin === getAuthDomain() ? (
-                                <Route path="/auth" element={<AuthPage />} />
+                                <Route path="/auth/*" element={<AuthPage />} />
```

Comment:
what happens if email verification is enabled, and the user visits the email verification screen when:
- tenantId is not stored in localstorage
- tenantId is stored in localstorage

what happens if the email verification link clicked screen is visited when:
- a session does not exist
  - tenantId is not stored in localstorage
  - tenantId is stored in localstorage
- a session exists
  - tenantId is not stored in localstorage
  - tenantId is stored in localstorage